addons/editor-theme3/forums/base.css

```diff
@@ -129,8 +129,7 @@ svg[class*="scratchblocks-style-scratch3"] .sb3-label,
   fill: var(--editorTheme3-commentColor);
   stroke: var(--editorTheme3-commentBorder);
 }
-.sb3-comment-label,
-.sb3-label.sb3-comment-label {
+svg[class*="scratchblocks-style-scratch3"] .sb3-label.sb3-comment-label {
   fill: var(--editorTheme3-commentText);
```

Comment:
The forum comment's text is pure black by default even without high contrast mode.

Reply:
scratchblocks still uses the 2.0 colors for comments, so I don't think the addon needs to match its default output exactly.